src/app/components/home/home-top-banner.component.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../styles/colors';
+@import '../../styles/dimensions';
+@import '../../styles/typography';
+
+.home-header-container {
+  display: flex;
+  background: linear-gradient($color-cerise-600, $color-primary-cerise);
```

Comment:
Properties should be ordered display, color, background

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)